Rakefile

```diff
@@ -120,3 +128,48 @@ namespace :qless do
   end
 end
 
+
+if ruby_meet_expectation("2.0")
```

Comment:
Your `ruby_meet_expectation` logic is nice and robust, but it seems like overkill to me -- this works just as well:

``` ruby
if RUBY_VERSION.to_f >= 2.0
end
```

We use this pattern _all over the place_ in RSpec to support a wide range of ruby versions (back to 1.8.7!) while leveraging features form newer ruby versions.  We've never had a problem with it.